src/frontend/js/hooks/useDashboardAddressForm.tsx

```diff
@@ -18,6 +18,19 @@ const messages = defineMessages({
   },
 });
 
+// / ! \ If you need to edit the validation schema,
+// you should also add/edit error messages above.
+const validationSchema = Yup.object().shape({
```

Comment:
I might be missing a point 😅 but why is it needed to nearly duplicate `validationSchema` that was already present in `validationSchema.ts`? I get that here we need a special shape to handle `is_main` boolean, but if so: why did you rename id from `save` to `is_main` so?